bfd/bfd-in2.h

```diff
@@ -3737,6 +3737,8 @@ pc-relative or some form of GOT-indirect relocation.  */
   BFD_RELOC_ARC_S21H_PCREL_PLT,
   BFD_RELOC_ARC_NPS_CMEM16,
   BFD_RELOC_ARC_JLI_SECTOFF,
+  BFD_RELOC_ARC_JLI_SECTOFF_SIMM12,
```

Comment:
Are those additional relocations? Why do we need them, I am not aware MWDT has them.